core/core.go

```diff
@@ -141,6 +141,10 @@ type Development struct {
 	Template string            `json:"-" yaml:"template,omitempty"`
 }
 
+func (d Development) UsesNext() bool {
```

Comment:
@t6d I mimicked  the other public functions in this module, but it's not having the effect I expect it to have. What's the correct way to expose this flag to the template engine so that I can use it from templates to conditionally change the content?

Reply:
You're looking at the wrong thing. `Template` is currently one of `javascript`, `javascript-react`, `typescript` or `typescript-react`. You need to do the `strings.Contains()` check against `Extension.Type`.